utxonursery.go

```diff
@@ -1745,13 +1729,17 @@ type kidOutput struct {
 	// transaction of the remote party.
 	absoluteMaturity uint32
 
+	// broadcastHeight is the earliest height at which this output could
+	// have been confirmed at.
+	broadcastHeight uint32
```

Comment:
I like the last option, as it should be safe to start watching the chain at the `Expiry` height.

Is it correct that `babyOutputs` will never have to be watched in-chain (and hence don't need their own height hint), since we just keep track of the current block height, and transition into a `kid` when the CLTV is reached?